SAR_Training/English/Master/ISCE3_Sentinel1_RTC.ipynb

```diff
@@ -0,0 +1,461 @@
+{
```

Comment:
<p>This block of code is dense. To make it easier to read I suggest something instead like</p><p> </p><p><code></code><code></p><p>config = config | {</p><p>    'runconfig': {</p><p>        'groups': {</p><p>             'input_file_group': {</p><p>                 'safe_file_path': str(data_dir_path/f"{scene}.zip"),</p><p>                 'orbit_file_path': str(orbitfile_path)</p><p>             },</p><p>             'dynamic_ancillary_file_group': {</p><p>                 'dem_file': str(merged_dem_path),</p><p>                 'dem_description': "Merged COP30 DEMS from s3://copernicus-dem-30m: {' '.join([p.split('/')[1] for p in dem_s3_paths])}"</p><p>             },</p><p>             'product_group': {</p><p>                 'output_dir': str(data_dir_path/'ISCE3_RTC'),</p><p>                 'save_mosaics': True,</p><p>                 'product_path': '.',</p><p>                 'scratch_path': str(data_dir_path/'scratch_dir'),</p><p>                 'product_id': f'OPERA_L2_RTC-S1_{scene}'</p><p>             },</p><p>             'processing': {</p><p>                 'polarization': 'dual-pol'</p><p>             }</p><p>       }</p><p>    } </p><p>}</p><p></code><code></code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ASFOpenSARlab/opensarlab-notebooks/pull/205/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>